docker-compose.yml

```diff
@@ -73,6 +73,14 @@ services:
     entrypoint: ['/wait-for-it.sh', '${DD_DATABASE_HOST}:${DD_DATABASE_PORT}', '-t', '30', '--', '/entrypoint-celery-beat.sh']
     environment:
       DD_DATABASE_URL: ${DD_DATABASE_URL}
+      DD_DATABASE_OPTIONS: # Boolean value, set if you are using the SSL Options
+      DD_DATABASE_OPTIONS_SSLMODE: # SSL Mode Options require, verify-full, or verify-ca
+                                   # Require needs SSLCERT, SSLKEY, and SSLROOTCERT Options
+                                   # Verify-Full needs SSLROOTCERT Option
+                                   # Verify-CA will verify the CA with public certifate signatures.
+      DD_DATABASE_OPTIONS_SSLCERT: # Path to SSL Cert
+      DD_DATABASE_OPTIONS_SSLKEY: # Path to SSL Key
+      DD_DATABASE_OPTIONS_SSLROOTCERT: # Path to SSL Root Cert
```

Comment:
Adding empty values to `docker-compose.yml` is not valuable.

Reply:
Thanks. These are just placeholders for the moment until I get documentation finished :-)